muesli/web/static/js/accidental-action-prevention.js

```diff
@@ -0,0 +1,23 @@
+function onActionClicked(event) {
+    let WAIT = "wait-for-confirmation";
+    let actionLink = event.target;
+
+    if(actionLink.classList.contains(WAIT)) {
+        // The action link has been already clicked, this click is the final confirm
+        return true;
+    } else {
+        actionLink.classList.add(WAIT);
+        actionLink.textContent = actionLink.textContent.trim()+" bestätigen"
+        return false;
+    }
+}
+
+function addAccidentalClickPreventionListener() {
+    for(action of document.getElementsByClassName("action")) {
+        if(action.tagName == "A" && action.classList.contains("needs-confirmation")) {
+            action.onclick = onActionClicked;
+        }
+    }
```

Comment:
Why do you go over each <a></a> having this class instead of adding the onclick event manually? Writing `<a onclick=accidentialClickPrevention class=needs-confirmation>` looks cleaner

Reply:
My intentions were that you only have to add `class="needs-confirmation"` to a button/link and the framework will do the rest for you.
Nevertheless, I can change the code to manual adding so we don't have to loop.

Reply:
I have performed the change as requested :)